examples/src/utils.py

```diff
@@ -0,0 +1,52 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np  
+from itertools import chain
+
+
+def plot_dfs(dfs, plot_func, dfs_cov: pd.DataFrame=None, when_crisis: pd.Series=None, legend: list=None, no_cols: int=3, figsize: tuple=(5, 2), **kwargs):
```

Comment:
Is it possible to use the "built-in" plotting functions? It'd be nice to avoid a utils file for an example notebook, and as I remarked in the notebook, if those functions don't do what you need them to do, I need to fix them up